Tests/SourceKitLSPTests/SyntaxRefactorTests.swift

```diff
@@ -42,6 +42,26 @@ final class SyntaxRefactorTests: XCTestCase {
     )
   }
 
+  func testAddDocumentationRefactorSingleParameter() throws {
+    try assertRefactor(
+      """
+        1️⃣func 2️⃣refactor(syntax: DeclSyntax) { }
```

Comment:
Is on the label valid? Parameter?

Reply:
It doesn't work in the `assertRefactor` tests because those invoke the refactoring directly from the syntax node. I did add a new code action test case that tests invoking documentation adding from the parameters (because that does the new logic of walking up parents to find the node on which to invoke the refactoring action).